### DISCUSSION TITLE
Show expressions styled as code?

### DISCUSSION THREAD

**Discussion by nicpottier** [nyaruka/floweditor#1107] [Ideas]
<img width="238" alt="Screen Shot 2019-04-19 at 4 09 19 PM" src="https://user-images.githubusercontent.com/168067/56444241-87e84d00-62bd-11e9-9398-78db244dabab.png">

**Comment by rowanseymour** [comment]
And perhaps also collapsed with tooltip?

<img width="361" alt="Captura de Pantalla 2019-04-22 a la(s) 12 45 43" src="https://user-images.githubusercontent.com/675558/56499057-92445a00-64fc-11e9-8dfa-e5126ad8a4b8.png">

**Comment by rowanseymour** [comment]
I don't think #531 is actually addressing this? @ericnewcomer

**Comment by ericnewcomer** [comment]
Oops. Tagged the wrong issue. Thanks.
